src/main/java/duke/util/Storage.java

```diff
@@ -0,0 +1,102 @@
+package duke.util;
+
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Deadline;
+import duke.task.Event;
+
+import java.io.IOException;
+import java.io.File;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.io.FileWriter;
+import java.util.ArrayList;
+
+/**
+ * The Storage class handles loading and saving of tasks to a file specified by a file path.
+ *
+ * @author Win Sheng
+ * @since 3 September 2023
+ */
```

Comment:
Overall, all classes have documentation along with constructors and methods! Well done 👍